test/__snapshots__/getConfig.spec.js.snap

```diff
@@ -62,15 +62,15 @@ exports[`validateConfig should not throw and should print validation warnings fo
 "
 WARN ● Validation Warning:
 
-  Unknown option \\"*.js\\" with value ['eslint --fix', 'git add'] was found in the config root.
-  
-  You are probably trying to mix simple and advanced config formats. Adding 
-  
-  \\"linters\\": {
-    \\"*.js\\": [\\"eslint --fix\\",\\"git add\\"]
-  }
-   
-  will fix it and remove this message.
+Unknown option \\"*.js\\" with value ['eslint --fix', 'git add'] was found in the config root.
```

Comment:
Is there a way to keep this indentation of 2 spaces?

Reply:
It is possible if we keep a header text:
```js
const message = dedent`
      WARNING
        Unknown option ${chalk.bold(`"${option}"`)} with value ${chalk.bold(
          format(config[option], { inlineCharacterLimit: Number.POSITIVE_INFINITY })
        )} was found in the config root.

        You are probably trying to mix simple and advanced config formats. Adding

        ${chalk.bold(`"linters": {
          "${option}": ${JSON.stringify(config[option])}
        }`)}

        will fix it and remove this message.
    `
```
Dedent uses the min of indentation to strip. So we need relative indentation. What do you suggest?

Reply:
Can we put 2 spaces on the first line:

```js
const message = dedent`  Unknown option ${chalk.bold(`"${option}"`)} with value ${chalk.bold(
          format(config[option], { inlineCharacterLimit: Number.POSITIVE_INFINITY })
        )} was found in the config root.

        You are probably trying to mix simple and advanced config formats. Adding

        ${chalk.bold(`"linters": {
          "${option}": ${JSON.stringify(config[option])}
        }`)}

        will fix it and remove this message.
    `
```

Reply:
Although it doesn't look very nice either...

Reply:
How about 
```js
const message = dedent`
  UNKNOWN OPTION
    Option ${chalk.bold(`"${option}"`)} with value ${chalk.bold(
      format(config[option], { inlineCharacterLimit: Number.POSITIVE_INFINITY })
    )} was found in the config root.

    You are probably trying to mix simple and advanced config formats. Adding

    ${chalk.bold(`"linters": {
      "${option}": ${JSON.stringify(config[option])}
    }`)}

    will fix it and remove this message.
`
```

Reply:
All caps are bad for readability. Also, I don't see a reason to have 2 titles. If dedent doesn't support our case, we should either not use it or add support to dedent. I don't want to compromise the UX in favor of better indented codebase.

Reply:
Okay. That makes sense. I'll look into `dedent`.

Reply:
@okonet Small update, I made a PR to `dedent` which would allow us to specify default indent - dmnd/dedent#14.